missions/HKR_Missions.txt

```diff
@@ -1168,10 +1431,7 @@ hakurei_missions_many_visitors_empire_slot = {
 		
 		provinces_to_highlight = {
 			religion = th_shinkism
-			NOT = { area = Makai_area }
-			NOT = { area = Makai_coast_state }
-			NOT = { area = Makai_peninsula_state }
-			NOT = { area = Western_makai_state }
+			region = makai_region
```

Comment:
Change doesn't seem to make sense from what I recall of her mission tree, what's your reasoning for it?

Reply:
oops on me misread the initial mission

Reply:
fixed